graphql/database/logs/UserEventLogModel.js

```diff
@@ -0,0 +1,52 @@
+import DynamoDBModel from '../base/DynamoDBModel'
+import types from '../fieldTypes'
+import tableNames from '../tables'
+import { permissionAuthorizers } from '../../utils/authorization-helpers'
+import { VALID_LOG_SCHEMAS, VALID_LOG_TYPES } from './logTypes'
+import { USER_EVENT_LOG } from '../constants'
+
+/*
+ * @extends DynamoDBModel
+ */
+class UserEventLogModel extends DynamoDBModel {
+  static get name() {
+    return USER_EVENT_LOG
+  }
+
+  static get hashKey() {
+    return 'userId'
```

Comment:
@kmjennison Would like your thoughts on this. If we happen to want to write two logs in the same mutation technically the hashKey/rangeKey could collide.

Reply:
I feel like something ideal would be event type / user id / timestamp; is there a way to express this that way?

Reply:
That said, I've elsewhere cut corners on the userId+timestamp index problem by assuming some small number of item creations will conflict and just making micro-adjustments to the timestamps:
https://github.com/gladly-team/tab/blob/master/graphql/database/users/logTab.js#L59-L63

Reply:
That's definitely a creative way to do it!